src/utils/json.ts

```diff
@@ -0,0 +1,48 @@
+// Regex to clean up added $refId & $ref properties, added by Realm.JsonSerializationReplacer
+// TODO: Investigate better solution.
```

Comment:
Will this be fixed before merging the PR? Perhaps you need to enable passing an option to the `JsonSerializationReplacer`?

Reply:
That was also my initial thought, but this is in place if it's decided not to extend `JsonSerializationReplacer`.